frontend/packages/client/src/pages/index.js

```diff
@@ -29,6 +30,9 @@ export default function AppPages() {
             <Route exact path="/about" component={About} />
             <Route exact path="/privacy-policy" component={PrivacyPolicy} />
             <Route exact path="/terms-of-service" component={TermsOfService} />
+            <Route exact path="/browse-communities">
```

Comment:
Made a question to Andrew about this URL. Not sure is there's a requirement for the url